src/01-functions.js

```diff
@@ -131,8 +160,10 @@ function compareTypes() {}
  *  flipSign(50);
  *  //> -50
  */
-function flipSign() {}
-
+function flipSign(value) {
```

Comment:
Again, we're hard coding a value here. The idea was to take the input value and if it was negative make it positive and vice versa. Unfortunately in this function you will only ever return the number 1 because you are reassigning the input `value` to be `Math.abs(-1)`(`Math` class should be capitalized) and returning `value`.